examples/debug/API/source-geojson.ipynb

```diff
@@ -1,4065 +0,0 @@
-{
```

Comment:
It would be nice to keep this debug notebook

Reply:
It has the same content as the geojson public example, if we need a debug example later to try more things we can add it then 👌

Reply:
:+1: